### PR TITLE
Some readme updates and a typo

### PR DESCRIPTION
**Pull request by dekvall**
- Added some more information to the build guide
 - Fixed a small typo in the create plugin script

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,57 +8,59 @@ provided "as is"; we make no guarantees about any plugin in this repo.
...
 ### Using the script
  1. Navigate to the folder in which you cloned the `plugin-hub` repository.
 
  2. Run the script with:
  ```
- python3 create_new_plugin.py
+ python3 create_new_plugin.py [--output_directory OUTPUT_DIRECTORY]
  ```
  It will ask you a series of questions, and then generate a folder with the name of your plugin.
+ There's also a non-interactive mode, you can list the options by running `python3 create_new_plugin.py --help`.
```
Comment (abextm) [inline README.md]: I don't think documenting the flags is particularly useful here, they are basically just so I can test it easily

File: README.md [hunk 2]
```diff
@@ -8,57 +8,59 @@ provided "as is"; we make no guarantees about any plugin in this repo.
 
 ## Creating new plugins
 There are two methods to create an external plugin, you can either:
+
  - Use [this](https://github.com/runelite/example-plugin/) plugin template.
 
  - Clone this repository and run the `create_new_plugin.py` script. This requires you to have `python3` installed
 
+**If you are using IntelliJ, you will need at least version 2017.3**
```
Comment (abextm) [inline README.md]: Is this really relevant in 2020?

Comment (dekvall) [reply README.md]: Somebody had trouble with it, so it thought there was no harm in adding the information.

File: README.md (hunk 1 above)
Comment (dekvall) [reply README.md]: I see, yea maybe not, i'll remove it.

**Pull request merged**